components/service/glean/src/main/java/mozilla/components/service/glean/Glean.kt

```diff
@@ -336,52 +329,21 @@ open class GleanInternalAPI internal constructor () {
      */
     fun handleBackgroundEvent() {
         // Schedule the baseline and event pings
-        sendPingsInternal(listOf(BASELINE_STORE_NAME, "events"))
-    }
-
-    /**
-     * Send a list of pings by name.
-     *
-     * Only custom pings (pings not managed by Glean itself) will be sent by
-     * this function. If the name of a Glean-managed ping is passed in, an
-     * error is logged to logcat.
-     *
-     * While the collection of metrics into pings happens synchronously, the
-     * ping queuing and ping uploading happens asyncronously.
-     * There are no guarantees that this will happen immediately.
-     *
-     * If the ping currently contains no content, it will not be sent.
-     *
-     * @param pingNames List of pings to send.
-     */
-    fun sendPings(pingNames: List<String>) {
-        val pingsToSend = pingNames.filter { pingName ->
-            if (BUILTIN_PINGNAMES.contains(pingName)) {
-                logger.error("Attempted to send built-in ping $pingName")
-                false
-            } else {
-                true
-            }
-        }
-
-        // Send pings is a "fire and forget" operation, we don't need to wait on
-        // it and we don't care about its return value.
-        sendPingsInternal(pingsToSend)
+        sendPings(listOf(Pings.baseline, Pings.events))
     }
 
     /**
-     * Send a list of pings by name.
+     * Send a list of pings.
      *
      * While the collection of metrics into pings happens synchronously, the
      * ping queuing and ping uploading happens asyncronously.
      * There are no guarantees that this will happen immediately.
      *
-     *
      * If the ping currently contains no content, it will not be sent.
      *
-     * @param pingNames List of pings to send.
+     * @param pings List of pings to send.
```

Comment:
I was missing this before, can you please also add a `@returns` here? we're returning a `Job` anyway.